bindings/ios/MEGABackgroundMediaUpload.h

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file MEGABackgroundMediaUpload.h
+ * @brief Time zone details
+ *
+ * (c) 2018 - by Mega Limited, Auckland, New Zealand
+ *
+ * This file is part of the MEGA SDK - Client Access Engine.
+ *
+ * Applications using the MEGA API must present a valid application key
+ * and comply with the the rules set forth in the Terms of Service.
+ *
+ * The MEGA SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * @copyright Simplified (2-clause) BSD License.
+ *
+ * You should have received a copy of the license along with this
+ * program.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MEGABackgroundMediaUpload : NSObject
+
+/**
+ * @brief Encrypt the file or a portion of it
+ *
+ * Call this function once with the file to be uploaded.  It uses mediainfo to extract information that will
```

Comment:
Extra spaces after .

Reply:
fixed

Reply:
this is copied from megaapi.h, I have notified @mattw-mega to remove the extra spaces as well. Thanks for catching this.